src/test.py

```diff
@@ -46,10 +46,9 @@ def test_with_image(img_path):
         cv2.readOpticalFlow(img, (0, 0), (width, height), (0, 0, 255), thicc)
 
     cv2.imshow('frame', img)
-
-    cv2.waitKey(0)
-
-    cv2.destroyAllWindows()
+    k = cv2.waitKey(0)
+    if k == 27:
+    	cv2.destroyAllWindows()
```

Comment:
I see some indentation issues here. Did this work well?

Reply:
it is working on mine